plugin-fancymenu/lxqtfancymenuappmap.h

```diff
@@ -121,11 +121,11 @@ class LXQtFancyMenuAppMap
 private:
     typedef QMap<QString, AppItem *> AppMap;
     AppMap mAppSortedByDesktopFile;
```

Comment:
Now `typedef` for `AppMap` can be removed and
```c++
    QMap<QString, AppItem *> mAppSortedByDesktopFile;
```

Reply:
Done.